apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -106,6 +106,28 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
+        invalidServicesError:  {
+            schemaDoc: 'Indicates if services are valid and add up to total sum toPay.',
+            type: Virtual,
+            isRequired: false,
+            resolver: async (receipt) => {
+                let servicesTotal = Big(0)
+                const toPay = Big(get(receipt, 'toPay'))
+
+                if (get(receipt, 'services')){
+                    for (let service of get(receipt, 'services')){
+                        servicesTotal = Big(get(service, 'toPay')).plus(Big(servicesTotal))
+                    }
+                }
+
+                const servicesAreValid = servicesTotal.cmp(toPay) === 0
+
+                if (!servicesAreValid) return `Services sum (${servicesTotal}) does not add up to the toPay (${toPay}) amount correctly`
```

Comment:
is it ok to have servicesAreValid = false if there are no services?

Reply:
thank you, fixed it so now if `services` are an empty array `invalidServicesError` will be null